### PR TITLE
Adding LICENSE

### PR DESCRIPTION
**Pull request by nate-double-u**
Adding LICENSE, LICENSE-docs files; updating README.

closes: https://github.com/etcd-io/website/issues/121

### CODE REVIEW

**Review by spzala — COMMENTED** [review]
lgtm
Thanks @nate-double-u

**Review by ptabor — APPROVED** [review]
Thank you.

**Comment by chalin** [comment]
@nate-double-u et al.: FYI, on the grpc.io site of things, we stream-lined the licence(s), mainly because our .io websites _are_ documentation repos (vs. code repos), and any code that is present in the website is usually there in support of the docs. I.e., grpc.io now have only one license file: https://github.com/grpc/grpc.io/blob/main/LICENSE

**Comment by nate-double-u** [comment]
> @nate-double-u et al.: FYI, on the grpc.io site of things, we stream-lined the licence(s), mainly because our .io websites _are_ documentation repos (vs. code repos), and any code that is present in the website is usually there in support of the docs. I.e., grpc.io now have only one license file: https://github.com/grpc/grpc.io/blob/main/LICENSE

I like the idea of combining the two licenses into 1 file, but doing it the way you've linked, aren't you missing the Apache license text?

I think for the site itself and any etcd code samples, the license is Apache, but the documentation is CC-BY-4.0 (which i think this setup reflects).

**Comment by chalin** [comment]
When we specify the licence and link to it, AFAIK we don't need to license text. It's provided as a convenience, not a necessity.

**Comment by nate-double-u** [comment]
good to know

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,408 @@
+
```
Comment (chalin) [inline LICENSE:1]: Delete this leading blank line.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+Except as otherwise noted, the content of this repository is licensed under the
+Creative Commons Attribution 4.0 License [1], and code and code samples are
+licensed under the Apache License, Version 2.0 [2].
+
+Copyright 2015-present etcd Authors. All rights reserved.
```
Comment (chalin) [inline LICENSE:6]: For etcd, I believe that the starting year is 2013.

**Review by chalin — APPROVED** [review]
Other than the two minor issues, this LGTM

**Pull request merged**